seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -24,18 +36,7 @@
         "reason": "It may be removed in a future release, please configure to 'seata.tx-service-group'."
       }
     },
-    {
-      "name": "seata.application-id",
-      "type": "java.lang.String",
-      "sourceType": "io.seata.spring.boot.autoconfigure.properties.SeataProperties",
-      "defaultValue": "${spring.application.name:}"
-    },
-    {
-      "name": "seata.data-source-proxy-mode",
-      "type": "java.lang.String",
-      "sourceType": "io.seata.spring.boot.autoconfigure.properties.SeataProperties",
-      "defaultValue": "AT"
-    },
+
     {
       "name": "seata.client.rm.async-commit-buffer-limit",
```

Comment:
Look like lack of tableMetaCheckerInterval

Reply:
+1